src/app/ubs-admin/components/ubs-admin-go-back-modal/ubs-admin-go-back-modal.component.scss

```diff
@@ -0,0 +1,31 @@
+.container {
+  margin: 0;
+  padding: 0;
+  font-family: var(--primary-font);
+  width: 422px;
+}
+
+.question {
+  font-size: 16px;
+  color: var(--ubs-primary-grey);
+  padding: 1px 30px 20px 10px;
+}
+
+.disagree {
+  font-weight: bold;
+  font-size: 14px;
+  border: 1px solid var(--primary-green);
+  background: #fff;
+  color: #13aa57;
+  padding: 8px 28px;
+  margin-right: 25px;
+}
+
+.agree {
+  font-weight: bold;
+  font-size: 14px;
+  border: 1px solid var(--primary-green);
+  background: #13aa57;
+  color: #fff;
+  padding: 8px 28px;
+}
```

Comment:
move this class to global scope
because you have same logic in 2 files

Reply:
done